webview/src/plots/components/templatePlots/NoPlotsToDisplay.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { StartButton } from '../../../shared/components/button/StartButton'
+import { EmptyState } from '../../../shared/components/emptyState/EmptyState'
+import { addPlot, selectPlots } from '../../util/messages'
+import { PlotsState } from '../../store'
+
+export const NoPlotsToDisplay: React.FC = () => {
+  const { hasUnselectedPlots } = useSelector(
+    (state: PlotsState) => state.webview
+  )
+  return (
+    <EmptyState isFullScreen={false}>
+      <p>No Plots to Display</p>
+      {hasUnselectedPlots ? (
+        <div>
+          <StartButton
+            isNested={true}
+            onClick={selectPlots}
+            text="Select Plots"
+          />
+          <StartButton
```

Comment:
The custom section "Add Plot" button is a bit different than the template one since it can take you to right to custom plot creation, unlike the data series which needs that extra description for more context.

We could keep as is, but if we wanted them to function the same we could:

1. Have the custom plot section open the "Select Plot Type" quick pick as well. Would mean an extra click for the user but would reduce some code. 
2. Update our "Add Data Series Plot" action to open a quick pick first instead of the file selector, allowing the user to get context and type in a relative file if preferred. Would get rid of the extra click a user needs to make, but would make data series plot creation a bit more complex.

Reply:
I vote for option 1.

Reply:
Yes, "Add Data Series" opens with the dvc.yaml being opened. Makes sense! I'll update the code to use the add plot command instead.